packages/twenty-front/src/modules/views/view-picker/components/ViewPickerDropdown.tsx

```diff
@@ -86,6 +91,11 @@ export const ViewPickerDropdown = () => {
           ) : (
             <IconList size={theme.icon.size.md} />
           )}
+          <PageTitle
+            title={`${
+              currentViewWithCombinedFiltersAndSorts?.name ?? 'All'
```

Comment:
Not mandatory but I think we could also add a `PageTitle` within RecordIndexPage to handle "All Companies - Companies" (or maybe just "Companies"). In practice that will always be overwritten by the title logic in `ViewBar` because in today's setup RecordIndexPage always has ViewBar as a child component but it could easily stop being the case (what if we want to have a read-only view that would not feature the ViewBar to edit it?).

Ideally all the records titling logic would be at the same place in RecordIndexPage using `useGetCurrentView` to fetch the view information but the hook is not usable as is in RecordIndexPage at the moment so let's not make things complex.